WPMedia/fusion-news-theme-blocks
# PEN-1506 Simple List & Card List has div instead of article tags

**If you have not filled out the checklist below, the pr is not ready for review.**

## Description
_Information about what you changed for this PR_
Relace div by using article tag for Simple list and card list
## Jira Ticket
- [PEN-1506](https://arcpublishing.atlassian.net/browse/PEN-1506)

## Acceptance Criteria
_copy from ticket_
Expected Behavior
article-tag wrapping an article for Simple list and card list

## Test Steps
- Add test steps a reviewer must complete to test this PR
1. In Fusion-News-Theme repo, checkout master & git pull
2. After checking out this feature branch in fusion-news-theme-blocks, run rm -rf ./node_modules && npx lerna clean && npm i && npx lerna bootstrap && cd blocks/header-nav-chain-block && npm i && cd ../..
3. Run npx fusion start -f -l @wpmedia/card-list-block,@wpmedia/simple-list-block in Fusion-News-Theme
4. Create a page then add Card List and Simple List for testing purpose
Inspect the html element on all of article to verify
## Effect Of Changes
### Before
_Example: When I clicked the search button, the button turned red._
![image](https://user-images.githubusercontent.com/61674931/101124293-50ace800-3629-11eb-8e96-a05e0d3027dc.png)

[include screenshot or gif or link to video, storybook would be awesome]

### After
_Example: When I clicked the search button, the button turned green._

## Simple List result
![image](https://user-images.githubusercontent.com/61674931/101124374-7d60ff80-3629-11eb-95d8-89510f20f729.png)

## Card List result
![image](https://user-images.githubusercontent.com/61674931/101124610-f9f3de00-3629-11eb-9599-861579468620.png)

[include screenshot or gif or link to video, storybook would be awesome]

## Dependencies or Side Effects
_Examples of dependencies or side effects are:_
- Additional settings that will be required in the blocks.json
- Changes to the custom fields which will require users to reconfigure features
- Update to css framework or SDK
- Dependency on another PR that needs to be merged first

## Review Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).
